SDWebImage.podspec

```diff
@@ -31,6 +31,7 @@ Pod::Spec.new do |s|
     core.source_files = 'SDWebImage/*.{h,m}', 'WebImage/SDWebImage.h', 'SDWebImage/Private/*.{h,m}'
     core.exclude_files = 'SDWebImage/MapKit/*.{h,m}'
     core.private_header_files = 'SDWebImage/Private/*.h'
+    core.prefix_header_contents = '#import "SDInternalMacros.h"'
```

Comment:
This pch is internal only. So does not port to any publc APIs or effect any other Pod or Application outside SDWebImage/Core.

Reply:
> This pch is internal only. So does not port to any publc APIs or effect any other Pod or Application outside SDWebImage/Core.

I'm creating a test pod. I suggest don't move the dispatch_main_async_safe and SD_LOCK macros into private pch, as I known some project using this macro to protect thread. One of my projects depend on  dispatch_main_async_safe macro.

Reply:
I don't think this `dispatch_main_async_safe` is a public API. It's not documented at all. 

Since this 5.0 is not released, there is the the last chance to polish API stable (or we have to wait 6.0). Maybe this need last check.

I also notice this, this is why I change this into a seperate commit. If so, I can split this MR into two, one does not move `dispatch_main_async_safe`, one does.

Reply:
I have discussed with other guys, I thought your opinion is right. And I created a test pod, tried `prefix_header_contents` way, then used `s.prefix_header_file`,  pod auto copy SDTestMacros.h's content to `SDTest-prefix.pch` file. I prefer `prefix_header_contents`.